Analysis/Tasks/PWGDQ/dileptonEE.cxx

```diff
@@ -320,20 +336,124 @@ struct DileptonEE {
         dileptonList(event, fValues[VarManager::kMass], fValues[VarManager::kPt], fValues[VarManager::kEta], fValues[VarManager::kPhi], -2, filter);
         for (int i = 0; i < fNTrackCuts; ++i) {
           if (filter & (uint8_t(1) << i)) {
-            fHistMan->FillHistClass(Form("PairsBarrelMM_%s", fTrkCutsNameArray->At(i)->GetName()), fValues);
+            fHistMan->FillHistClass(Form("PairsBarrelLSnn_%s", fTrkCutsNameArray->At(i)->GetName()), fValues);
           }
         }
       }
     }
   }
 };
 
+struct EventMixing {
+  OutputObj<THashList> fOutputList{"output"};
+  HistogramManager* fHistMan;
+  float* fValues;
+  // NOTE: The track filter produced by the barrel track selection contain a number of electron cut decisions and one last cut for hadrons used in the
+  //           dilepton - hadron task downstream. So the bit mask is required to select pairs just based on the electron cuts
+  uint8_t fTwoTrackFilterMask = 0;
```

Comment:
You don't need this mask in your case, since the track filter contains just the electron cuts

Reply:
done